src/maintainer/infrastructure.rst

```diff
@@ -281,35 +281,75 @@ Compilers and Runtimes
 
 Conda-forge builds and maintains its own set of compilers for various languages
 and/or systems (e.g., ``C``, ``FORTRAN``, ``C++``, ``CUDA``, etc.). These are used
-in all of our CI builds to build both core dependencies (e.g., ``Python``) and maintainer-contributed
-packages. While we do not have any formal policies or promises of support for these
-compilers, we have historically maintained them according to the following (non-binding)
-principles.
-
-* The authoritative source of the current compilers and versions for various languages
-  and platforms is the `conda_build_config.yaml <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`_
-  in the `conda-forge/conda-forge-pinning-feedstock <https://github.com/conda-forge/conda-forge-pinning-feedstock>`_
-  as described in :ref:`globally_pinned_packages`.
-* We provide no support of any kind in terms of the long-term stability of these pinnings.
-* We upgrade them in an ad-hoc manner on a periodic basis as we have the time and energy to do so.
-  Note that because of the way we enforce runtime constraints, these compiler upgrades will not break
-  existing packages. However, if you are using the compilers outside of ``conda``, then you may find issues.
-* We generally provide notice in the form of an announcement when a compiler is going to be upgraded.
-  Note that these changes take a bit of time to complete, so you will generally have time
-  to prepare should you need to.
-* Some of the criteria we think about when considering a compiler migration include
-  1) the degree of disruption to the ecosystem, 2) the amount of work for the ``core`` team,
-  and 3) the amount of time it will cost our (volunteer) feedstock maintainers.
```

Comment:
A lot of the language here was agreed upon by the core team and should stay.

Reply:
Sure. I've put them back in with the exception of the last two, which aren't relevant anymore, since we don't have to migrate to change the compiler version.

Reply:
Though I'd argue that even the middle two points are not relevant anymore either (and the first one I had kept in the running text).